UnityProject/Assets/Scripts/Core/Database/ServerData.ServerStatus.cs

```diff
@@ -96,9 +96,20 @@ IEnumerator SendServerStatus()
 				{
 					string s = req.GetResponseHeader("set-cookie");
 					hubCookie = s.Split(';') [0];
-					req = UnityWebRequest.Get("http://ipinfo.io/ip");
-					yield return req.SendWebRequest();
-					publicIP = Regex.Replace(req.downloadHandler.text, @"\t|\n|\r", "");
+					if (string.IsNullOrEmpty(config.PublicAddress) == false)
+					{
+						publicIP = config.PublicAddress;
+					} 
+					else if (string.IsNullOrEmpty(config.BindAddress) == false)
+					{
+						publicIP = config.BindAddress;
+					}
+					else
+					{
+						req = UnityWebRequest.Get("http://ipinfo.io/ip");
+						yield return req.SendWebRequest();
+						publicIP = Regex.Replace(req.downloadHandler.text, @"\t|\n|\r", "");
```

Comment:
by convention, always add a comment explaining the regex

Reply:
Not my code

Reply:
`/\t|\n|\r/` -> Match Tab, Newline, or Carriage Return.

Reply:
I understand you just moved it, but can you improve it and add the comment, pretty please?